prime-router/build.gradle.kts

```diff
@@ -26,23 +26,23 @@ import java.io.FileInputStream
 import java.time.LocalDateTime
 import java.time.format.DateTimeFormatter
 import java.util.Properties
-import kotlin.collections.mutableMapOf
 
 plugins {
-    kotlin("jvm") version "1.7.0"
+    kotlin("jvm") version "1.6.21"
     id("org.flywaydb.flyway") version "8.5.13"
     id("nu.studer.jooq") version "7.1.1"
     id("com.github.johnrengelman.shadow") version "7.1.2"
     id("com.microsoft.azure.azurefunctions") version "1.8.2"
     id("org.jlleitschuh.gradle.ktlint") version "10.2.1"
     id("com.adarshr.test-logger") version "3.2.0"
     id("jacoco")
-    id("org.jetbrains.dokka") version "1.7.0"
+    id("org.jetbrains.dokka") version "1.6.20"
     id("com.avast.gradle.docker-compose") version "0.16.4"
+    id("org.jetbrains.kotlin.plugin.serialization") version "1.6.21"
 }
 
 group = "gov.cdc.prime"
-version = "0.2-SNAPSHOT"
+version = "0.1-SNAPSHOT"
```

Comment:
We are working with 0.2-SNAPSHOT, now.  Perhaps, your code is not up to date.

Reply:
I looked through your code.  Perhaps, they need to be rebased with the master branch.

Reply:
fixed